docs/pages/reference/networking.mdx

```diff
@@ -36,21 +36,21 @@ following use cases:
 </TabItem>
 <TabItem scope={["cloud", "team"]} label="Cloud-Hosted">
 
-All Teleport services (e.g., the Application Service and Database Service) have
-an optional `public_addr` property that you can modify in each service's
-configuration file. The public address can take an IP or a DNS name. It can also
-be a list of values:
+On Teleport Enterprise (Cloud), you can choose the sub-domain of
+the domain `teleport.sh` for your account. That fully qualified domain name
+(e.g., `example.teleport.sh`) is managed by Teleport for your account 
+along with any sub-domains assigned to Teleport-protected applications (e.g., `grafana.example.teleport.sh`).
+
+The public address (`public_addr`) for the Teleport Application Service is configurable. In the case of web applications, the public address must be a subdomain of the Teleport account URL
```

Comment:
Is this the public_addr for the Application Service or for the app itself?

Reply:
The app

Reply:
Ok, I'm still not quite happy with this.

This page is about setting the public_addr for agents, which really doesn't matter when you use reverse tunnels (as is required on cloud).

We've taken a page about setting the public_addr for agents and added an entirely new concept (application access), which adds confusion IMO.

Reply:
Let's do that then. We can say something like:

In Cloud-Hosted Teleport installations, agent services always connect using reverse tunnels, so there is no need to set a public address for the agent.

Reply:
Please see update.